plugins/balloontoolbar/plugin.js

```diff
@@ -723,6 +736,16 @@
 
 				CKEDITOR.ui.balloonPanel.prototype.attach.call( this, element, options );
 			};
+
+			// We don't want to expose this method as it should be called only in very specific moments:
```

Comment:
This is why you should make it a private method (please don't forget to include `@since` information).

As a result you won't need to do things like `CKEDITOR.ui.balloonPanel.prototype.deregisterFocusable.call( this, element );` - the code will be simplified.